src/annoylib.h

```diff
@@ -488,6 +492,10 @@ struct Hamming : Base {
   static inline T pq_initial_value() {
     return numeric_limits<T>::max();
   }
+  template<typename T>
+  static inline int std_popcount(T v) {
```

Comment:
```suggestion
  static inline int std_popcount(T v) {
    // Note: this is only used with Visual Studio
```

Reply:
might want to add a comment like ^

Reply:
Looks good. I'm unclear on whether this will use intrinsics or not so this make have a performance impact.